zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/config/asynchttpclient/package.scala

```diff
@@ -15,22 +15,17 @@ package object asynchttpclient {
   /** An [[SttpClient]] layer configured with the proper SSL context based on the provided
     * [[K8sClusterConfig]] using the async-http-client-backend-zio backend.
     */
-  val k8sSttpClient: ZLayer[K8sClusterConfig with System with Any, Throwable, SttpClient] = {
-    (config: K8sClusterConfig) =>
-      (for {
-        runtime                    <- ZManaged.runtime[Any]
-        sslContext                 <- SSL(config.client.serverCertificate, config.authentication).toManaged
-        disableHostnameVerification = config.client.serverCertificate match {
-                                        case K8sServerCertificate.Insecure => true
-                                        case K8sServerCertificate.Secure(
-                                              _,
-                                              disableHostnameVerification
-                                            ) =>
-                                          disableHostnameVerification
-                                      }
-        client                     <-
-          ZManaged
-            .acquireReleaseAttemptWith(
+  val k8sSttpClient: ZLayer[K8sClusterConfig with Scope with Any, Throwable, SttpClient] = ZLayer {
```

Comment:
The layer should not require `Scope` as an input - rather you should define the layer with `ZLayer.scoped { ... }` which provides a `Scope` for the layer's lifetime. That's equivalent of the previous code of constructing the layer from a `ZManaged`.

Also you should remove `with Any` as it does not add anything.